compiler/rustc_save_analysis/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 #![recursion_limit = "256"]
 #![allow(rustc::potential_query_instability)]
 #![feature(never_type)]
+#![feature(associated_type_defaults)]
```

Comment:
Please do not use associated type defaults, r-a chokes on them, https://github.com/rust-lang/rust-analyzer/issues/13693.